reference-artifacts/SAMPLE_CONFIGS/config.example.json

```diff
@@ -1863,7 +1863,7 @@
             }
           }
         ],
-        "xxfirewall-manager": {
+        "firewall-manager": {
```

Comment:
We have found the firewall manager appliance is not being used by most customers and was simply costing them $$$.  I'd prefer to leave this out of the default sample.  Customers can opt-in during install or on subsequent SM runs.

Reply:
And I thought did my daughter(2)  added these xx/zz :). Will add back